.stylelintrc

```diff
@@ -8,6 +8,7 @@
   ],
   "rules": {
     "at-rule-disallowed-list": ["each", "function"],
+    "scale-unlimited/declaration-strict-value": "never",
```

Comment:
We're mixing our SCSS stylelint rules with stylus in Vuepress. Gotta clean that up at some point.